docs/gl_objects/groups.rst

```diff
@@ -288,6 +288,15 @@ ancestor groups)::
 
     members = group.members_all.list(get_all=True)
 
+List all groups and all group member with access level::
```

Comment:
This feels like it's covering a bit more than just the group members API and almost like a small cookbook example. WDYT about narrowing it down to just the API calls for members, or moving this example to a separate documentation page for longer examples?